doc/usage/QuickStart.ipynb

```diff
@@ -0,0 +1,542 @@
+{
```

Comment:
Please remove this file (it's a local, temporary copy) and put a `.gitignore` file:
```gitignore
*.ipynb
```
under the `doc` folder (!), so that only `ipynb` files under this folder are ignored.